### PR TITLE
Log failures from sanity stage

### PR DESCRIPTION
**Pull request by happz**
This was mistakenly omitted during refactoring.

### CODE REVIEW

File: gluetool/glue.py [hunk 1]
```diff
@@ -603,21 +603,29 @@ def _for_each_module(self, modules, callback, *args, **kwargs):
...
+    def _log_failure(self, module, failure, label=None):
+        # type: (Module, Failure, Optional[str]) -> None
         """
         Log a failure, and submit it to Sentry.
 
         :param Module module: module to use for logging - apparently, the failure appeared
             when this module was running.
         :param Failure failure: failure to log.
-        :param str label: label for loggign purposes.
+        :param str label: label for loggign purposes. If it's set and exception exists, exception
```
Comment (thrix) [inline gluetool/glue.py]: typo; loggign

File: gluetool/glue.py [hunk 2]
```diff
@@ -603,21 +603,29 @@ def _for_each_module(self, modules, callback, *args, **kwargs):
...
+            message is appended. If it's not set, failure exception message If failure has no exception,
```
Comment (thrix) [inline gluetool/glue.py]: missing dot before If

**Review by thrix — CHANGES_REQUESTED** [review]
Only nitpicks ... otherwise LGTM

File: gluetool/glue.py (hunk 1 above)
Comment (happz) [reply gluetool/glue.py]: Fixed.

File: gluetool/glue.py (hunk 2 above)
Comment (happz) [reply gluetool/glue.py]: Fixed.

**Comment by thrix** [comment]
manual testing looks fine, I now see the error:
```
[10:29:51] [E] [pipeline-install-ancestors] Missing required 'playbook' option
```

**Review by marmolak — APPROVED** [review]

**Pull request merged**